ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml

```diff
@@ -7,17 +7,18 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
-    - name: OS_CLOUD
     - name: OPENSTACK_EXTERNAL_NETWORK
-    - name: OPENSTACK_COMPUTE_FLAVOR
-    - name: BASE_DOMAIN
+      default: "external"
+    - name: OPENSTACK_INSTANCE_FLAVOR
+      default: "m1.s2.xlarge"
```

Comment:
added default values here so they don't need to be defind in the workflow, but they can still be defined there if needed.